src/client/swagger/unity.yml

```diff
@@ -300,23 +336,23 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
-  '/orgs/{orgID}/invites/{inviteID}':
+  '/orgs/{orgId}/invites/{inviteId}':
     delete:
-      operationId: DeleteOrgsIDInviteID
+      operationId: DeleteOrgsIdInviteId
```

Comment:
It might be worth noting in the response that you will get a `404` from the delete on a retry (not idempotent) or if the org id doesn't match. This Quartz behavior differs from IDPE behavior where @glinton has structured deletes to be idempotent.

Reply:
@randycoulman thoughts?

Reply:
My initial thought is that a 404 is the right response if the org ID is missing. But if there is no invite with that ID, then we should return a successful response in order to make this idempotent.

Reply:
Agreed, I'll update our issue to reflect retry should return a successful response.